.github/workflows/ciab.yaml

```diff
@@ -45,6 +45,7 @@ on:
       - 'NOTICE'
       - 'traffic_control/java/**'
       - 'cache-config/testing/**'
+        'cache-config/testing/docker/trafficserver/**'
```

Comment:
Since this section is `on.push.paths-ignore`, `cache-config/testing/docker/trafficserver/**` should be prefixed with `!` to register the exception.

Reply:
@zrhoffman I guess I mis-understood your ask ^^^ above ^^^.  What precisely do you want added/not added to ciab.yaml

Reply:
Just making that line
```yaml
        '!cache-config/testing/docker/trafficserver/**'
```
to trigger the *CDN in a Box CI* workflow when that path is modified. It isn't explicitly related to this PR, so we can just as easily add it in a separate PR after (or before) #5938 is merged.

Reply:
I'm going to revert the ciab.yaml changes.  Update it in another PR.